docs/install.rst

```diff
@@ -63,14 +63,32 @@ packages install config files and systemd unit files. The dashboard is installed
 package.
 
 
-With pip
----------
-Inmanta can be installed with pip.
+Linux, Mac and Windows with pip install
+---------------------------------------
+Inmanta published new releases on `PyPI <https://pypi.org/project/inmanta/>`_. You can use the built-in python package manager
```

Comment:
weird bit of explanation, we don't explain we have RPM repo's either.

I would keep it short: to install using pip, use the following command. 
(virtual env is in there, people who understand it can use it or not, people who don't  will use it)